src/components/SlideCard.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <v-card>
+    <v-container
+      fluid
+      grid-list-lg>
+      <v-layout row>
+        <v-flex xl7>
+          <div class="headline">
+            {{ title }}
+          </div>
+        </v-flex>
+        <v-flex xs12>
+          <v-card-media
+            :src="imageUrl"
+            height="150px"
+            contain/>
+        </v-flex>
+      </v-layout>
+    </v-container>
+    <v-card-text class="overflow">
+      {{ description.substring(0, 50).concat('...') }}
```

Comment:
Ehtasham, I know I had this on the other branch, would you mind abstracting this into a method (a computed property will be much better).

Reply:
The same applies to the title too, as it may need to be truncated as well

Reply:
Done, added a screenshot for it as well.